scripts/update-workflow-issue

```diff
@@ -18,6 +18,21 @@ list_issues_and_prs() {
   (gh pr list -A "$user" "$@" --json number,labels,mergedAt | jq '[.[] | .["closedAt"] = .mergedAt | del(.mergedAt)]'; gh issue list -a "$user" "$@" --json number,labels,closedAt) | jq '.[]' | jq -rs 'sort_by(.closedAt) | reverse' | jq -r "$(map_to_markdown)"
 }
 
+list_prioritized_issues() {
+  gh issue list "$@" --json number,labels | jq '.[]' | jq -s '
+    map(
+      . as $issue |
```

Comment:
Wew that is some deep jq mojo @worstell 👏

Related...I'm wondering if maybe we should rewrite this script in Python at some point 😂

Reply:
you can thank chatgpt

Reply:
Thanks Chad!